dadapy/metric_comparisons.py

```diff
@@ -276,6 +282,8 @@ def greedy_feature_selection_full(self, n_coords, k=1, n_best=10, symm=True):
             best_tuples (list(list(int))): best coordinates selected at each iteration
             best_imbalances (np.ndarray(float,float)): imbalances (full-->coords, coords-->full) computed at each
                 iteration, belonging to the best tuple
+            all_imbalances (list(list(list(int)))): all imbalances (full-->coords, coords-->full), computed
+              at each iteration, belonging all greedy tuples
```

Comment:
same problem here, the space before "at each ..." should be the same as the space before "iteration" at line 284. These spacings are important to correctly compile the documentation on the website